test/arrays/vass/initialization-mapped-1.compopts

```diff
@@ -0,0 +1 @@
+--devel
```

Comment:
What is the motivation for adding --devel ?

Reply:
So that we print ``Block`` or ``DefaultDist`` or whatever instead of the cleaner ``domain(1, int ,false)``.